src/containers/Review/ReviewPageWrapper.tsx

```diff
@@ -66,6 +82,15 @@ const ReviewPageWrapper: React.FC = () => {
     })
   }
 
+  const validateReviewHandler = (): boolean => {
+    const { userId } = currentUser as User
+    const invalidSection = validateReview({ userId, reviewSections })
+    console.log(invalidSection)
+
+    setInvalidSection(invalidSection)
+    return invalidSection === undefined
+  }
+
   const submitResponseHandler = (_: any) => {
```

Comment:
```suggestions
  const submitResponseHandler = () => {
```
I don't think you need any parameter here at all.

Reply:
Thanks! Done